src/layout/Footer.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { FaFacebook, FaInstagram, FaTwitter, FaLinkedin } from 'react-icons/fa';
+
+// interface for social media links
+interface SocialLink {
+    name: string;
+    url: string;
+    icon: React.ReactNode;
+}
+
+// interface for Footer props
+interface FooterProps {
+    brandLogo:string;
+    socialLinks: SocialLink[];
+}
+
+const currentYear = new Date().getFullYear();
+
+const Footer: React.FC<FooterProps> = ({ brandLogo, socialLinks }) => {
```

Comment:
I think using `React.FC<FooterProps>` is not necessary as we don't need to pass any props to the Footer component.